lib/widgets/login.dart

```diff
@@ -223,22 +223,22 @@ class _AddAccountPageState extends State<AddAccountPage> {
   }
 }
 
-class EmailPasswordLoginPage extends StatefulWidget {
-  const EmailPasswordLoginPage({super.key, required this.serverSettings});
+class UsernamePasswordLoginPage extends StatefulWidget {
```

Comment:
Let's call this `PasswordLoginPage`.

In general, the phrase "password login" is more common for this kind of login flow than "username/password login". And in the Zulip API (https://zulip.com/api/get-server-settings), we call this auth method simply `password`.